components/Button.jsx

```diff
@@ -1,15 +1,20 @@
 import Link from "next/link";
 
-const Button = ({ classes, children, href }) => {
-  return (
-    <Link href={href}>
+const Button = (props) => {
+  return <Link href={props.href}>
       <p
-        className={`cursor-pointer rounded-full text-white border-4 border-white text-3xl font-bold p-4 bg-[#7055FD] ${classes}`}
+        className={`cursor-pointer rounded-full text-white border-4 ${props.borderColor} text-3xl font-bold ${props.padding} ${props.buttonBackground} ${props.classes}`}
```

Comment:
Instead of adding specific props for `borderColor`, `padding`, and `buttonBackground`, can we just add these as a single string to the `prop.classes` object if you want to overwrite it? I might be wrong, but I believe everything in classes will overwrite the existing defaults in the className.